app/controllers/admin/group_users_controller.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 module Admin
-  class GroupUsersController < AdminController
```

Comment:
why a base class change here?

Reply:
I _think_ because `AdminController` has a `before_action` that runs `authorize! :read, :admin_dashboard`, which was too broad of a permission check. A user could have permissions to edit a group's users without being able to see the entire admin dashboard. 

Is this the right approach? The group routes are still namespaced under `admin`, so there is some incongruity here

Reply:
I think its fine for the moment. that explanation makes sense to me.